config/detekt.yml

```diff
@@ -106,17 +106,19 @@ formatting:
   active: true
   android: false
   autoCorrect: true
+  ArgumentListWrapping:
+    active: false
   AnnotationOnSeparateLine:
     active: false
     autoCorrect: true
   ChainWrapping:
-    active: true
+    active: false
     autoCorrect: true
   CommentSpacing:
     active: true
     autoCorrect: true
-  EnumEntryNameCase: 
-    active: true
+  EnumEntryNameCase:
+    active: false
```

Comment:
Isn't the general convention?
https://kotlinlang.org/docs/reference/coding-conventions.html#enumerations

Reply:
actually will re-enable